### DISCUSSION TITLE
Cant get the hammer to work?

### DISCUSSION THREAD

**Discussion by Beesinthetrees** [Hydraxous/UltraFunGuns#58] [General]
Loving the idea of a big hammer, though it doesnt seem to working for me left mouse or "fire 1" just makes him like "prep" a swing as in lifting the hammer though he never actually swings, holding makes him lift it until it reaches its highest point tho releasing left mouse only returns to default position. Am i missing something?

**Comment by Beesinthetrees** [comment]
![Uploading hammerlift.gif…]()

**Comment by LoganTheAroAce** [comment]
im pretty sure that it takes longer to charge than it does to reach the highest point

**Comment by Hydraxous** [comment]
The charge time is broken, will be fixed in the next update. However a temporary solution is to change the weapons parameters in the Configgy menu for it.
